packages/flutter/lib/src/material/radio.dart

```diff
@@ -33,18 +37,306 @@ enum _RadioType { material, adaptive }
 const double _kOuterRadius = 8.0;
 const double _kInnerRadius = 4.5;
 
+/// A group for [Radio]s.
+///
+/// This widget treats all [Radio]s in the sub tree with the same type T as a
+/// group.
+///
+/// This widget handles the group value for the [Radio]s in the subtree with the
+/// same value type. To use this widget, the [Radio.groupValue] must leave as
+/// null. Otherwise, an assertion error is thrown.
```

Comment:
```suggestion
/// same value type. To use this widget, the [Radio.groupValue] must remain
/// null, for the RadioGroup to manage. Otherwise, an assertion error is thrown.
```